src/routes/docs/tutorials/vue-auth/step-3/+page.markdoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: tutorial
+title: Set up Appwrite
+description: Import and configure a project with Appwrite Cloud and Vue.js.
+step: 3
+---
+
+# Create project {% #create-project %}
```

Comment:
Can we just reuse step 2-3 from the vue tutorial. They don't need to be different other than description.